internal/app/export_test.go

```diff
@@ -27,7 +29,13 @@ func (m *MockAuthorizer) IsSenderAllowed(ctx context.Context, action string, sen
 }
 
 // Apply is a mock implementation of proxier, returning an error if requested in the mock.
-func (m *MockProxy) Apply(ctx context.Context, _, _, _, _, _, _ string) error {
+func (m *MockProxy) Apply(_, _, _, _, _, _ string) error {
+	m.ApplyCount++
```

Comment:
no need to mutex as it would have been detected when running the tests then (it would be in the obvious pass of the multi concurrency test). Let’s go with this :)